lib/capybara/node/element.rb

```diff
@@ -510,6 +511,11 @@ def evaluate_async_script(script, *args)
         JS
       end
 
+      ##
```

Comment:
Please don't document this, and instead mark it as `@api private` - It's not really intended for outside use